src/marine/glider_S2ioda.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+#
+# (C) Copyright 2019 UCAR
```

Comment:
2019-2020

Reply:
@guillaumevernieres 
1. will work on this, thanks for checking on what was neglected in the scripts.
2. in-situ T and S has profile identifier whereas AOML glider data don't, only named "temperature", "salinity", thought was easier to write T S separately at my first shot, but I will work on merging together
3. I agree but glider data do not have obs error, I assigned constant error , I could calculate standard deviation and assign if you like
4. will do
5. already done
6. waiting on @srherbener comment